content/en/blog/_index.md

```diff
@@ -1,3 +1,10 @@
 ---
-title: Blog
+title: Kubernetes Blog
+linkTitle: Blog
+menu:
+  main:
+    title: "Blog"
+    weight: 40
+    post: >
```

Comment:
@bep Can you point to a reference for `post: >` syntax? 

Otherwise I'm happy to see this text separated out into individual files as you've done.

Reply:
https://yaml-multiline.info/

Reply:
`pre` and `post` are just free-form fields on the menu item struct in Hugo, see: https://gohugo.io/content-management/menus/#add-non-content-entries-to-a-menu

Reply:
If you mean the "multiline YAML syntax", see https://yaml-multiline.info/

Reply:
> https://yaml-multiline.info/

Thanks, that's handy! 💯